openshift-serverless-clients.spec

```diff
@@ -4,7 +4,7 @@
 %global product_name OpenShift Serverless
 %global golang_version 1.14
 %global kn_version 0.18.4
-%global kn_release 1
+%global kn_release 2
 %global kn_cli_version v%{kn_version}
```

Comment:
@warrenvw : do we need to bump golang_version to specific one here?
current at line no 5: `golang_version 1.14`

Reply:
If the specific version is required, then you can use golang_version 1.14.12